design/search.go

```diff
@@ -56,6 +56,9 @@ var _ = a.Resource("search", func() {
 			a.Param("q", d.String, "Text to match against Name or description")
 			a.Param("page[offset]", d.String, "Paging start position")
 			a.Param("page[limit]", d.Integer, "Paging size")
+			a.Param("sort", d.String, func() {
+				a.Enum("name", "owner", "created", "-name", "-owner", "-created")
```

Comment:
Why not support `updated_at` and `-updated_at` as possible sort values? And why not have `created` renamed to `created_at`?

Reply:
`created` seems more concise than `created_at`, also cause in general reading terms you can say:

>sort as they were `created`

Reply:
Replied about `updated_at` above